### PR TITLE
[REBASED ON TOP OF 572] [WIP] tests: Adapt to CoroboTestCase 

### PR DESCRIPTION
**Pull request by nvzard**
Adapts tests to use CoroboTestCase after
release of Errbot 6.0.0a.

### CODE REVIEW

File: tests/corobo_test_case.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+
+class CoroboTestCase(FullStackTest):
+
+    def setUp(self,
+              extra_plugin_dir=None,
+              extra_test_file=None,
+              loglevel=logging.DEBUG,
+              extra_config=None,
+              plug_filename=None,):
```
Comment (jayvdb) [inline tests/corobo_test_case.py]: useless trailing comma.

File: tests/corobo_test_case.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+from errbot.templating import add_plugin_templates_path
+from pathlib import Path
+
+import logging
+
+
+class CoroboTestCase(FullStackTest):
+
+    def setUp(self,
+              extra_plugin_dir=None,
```
Comment (jayvdb) [inline tests/corobo_test_case.py]: I think you can remove all these args.

They are not being used, and unittest and pytest do not use them.

**Comment by jayvdb** [comment]
requests at https://github.com/coala/corobo/pull/581#pullrequestreview-137563815 and the following one were not done.

**Comment by nvzard** [comment]
@jayvdb because I've rebased this commit over the PR which updates errbot i.e. https://github.com/coala/corobo/pull/572, I made the changes in that commit.

**Comment by jayvdb** [comment]
https://github.com/coala/corobo/pull/572 wasnt split, so it included this patch also.

**Pull request closed without merging**